application.py

```diff
@@ -1145,6 +1152,121 @@ def __repr__(self):
         return f"User('{self.id}', '{self.username}', '{self.email}')"
 
 
+class UserModelView(ModelView):
+    """
+    User view
+    """
+    def is_accessible(self):
+        """
+        check if user can access admin page
+        :return:
+        """
+        return current_user.is_authenticated and current_user.user_type == "admin"
+
+    def inaccessible_callback(self, name, **kwargs):
+        """
+        return 403 page if not access admin page
+        :param name:
+        :param kwargs:
+        :return:
+        """
+        return self.render('errors/403.html')
+
+
+class OAuthUserView(BaseView):
+    """
+    OAuth User view
+    """
+    @expose('/')
+    def index(self):
+        """
+        can't access this page
+        :return:
+        """
+        return self.render('errors/403.html')
+
+    def is_accessible(self):
+        """
+
+        :return:
+        """
+        return current_user.is_authenticated and current_user.user_type == "admin"
+
+    def inaccessible_callback(self, name, **kwargs):
+        """
+
+        :param name:
+        :param kwargs:
+        :return:
+        """
+        return redirect(url_for('login'))
+
+
+class StrategiesView(BaseView):
+    """
+    Strategies view
+    """
+    @expose('/')
+    def index(self):
+        """
+
+        :return:
+        """
+        return self.render('errors/403.html')
+
+    def is_accessible(self):
+        """
+
+        :return:
+        """
+        return current_user.is_authenticated and current_user.user_type == "admin"
+
+    def inaccessible_callback(self, name, **kwargs):
+        """
+
+        :param name:
+        :param kwargs:
+        :return:
+        """
+        return redirect(url_for('login'))
+
+
+class HomePageView(BaseView):
+    """
+    Home page view
+    """
+    @expose('/')
+    def index(self):
+        """
+
+        :return:
+        """
+        return self.render('welcome.html')
+
+    def is_accessible(self):
+        """
+
+        :return:
+        """
+        return current_user.is_authenticated and current_user.user_type == "admin"
+
+    def inaccessible_callback(self, name, **kwargs):
+        """
+
+        :param name:
+        :param kwargs:
+        :return:
+        """
+        return redirect(url_for('login'))
+
+
+# admin
+admin = Admin(application)
+admin.add_view(HomePageView(name='Backtesting Platform', endpoint='home'))
+admin.add_view(UserModelView(User, db.session))
+# admin.add_view(OAuthUserView(name='OAuth User'))
+# admin.add_view(StrategiesView(name="Strategies"))
+
 if __name__ == "__main__":
-    # application.run(debug=False, threaded=True, host='0.0.0.0', port='5000')
+    # application.run(debug=True, threaded=True, host='0.0.0.0', port='5000')
```

Comment:
👏

Reply:
@jialiaz3 please remove commented code

Reply:
> what are these static files `static/profile_pics/1a909186a337b40a.jpg`? btw, I really like your alembic usage here, great work! 👏

These static files I removed are previous user images for testing.

Reply:
> @jialiaz3 please remove commented code

ok